chosen-docs/upstream/deploying-a-devfile-using-odo-on-IBM-Z-and-Power.md

```diff
@@ -0,0 +1,22 @@
+---
+# Page settings
+layout: default
+keywords:
+comments: false
+
+# Hero section
+title: Title here
+description: Description here
+
+# Micro navigation
+micro_nav: true
+
+# Page navigation
+page_nav:
+    prev:
+        content: Previous page
```

Comment:
content should be "Deploying your first application using odo"
and url "/docs/deploying-a-devfile-using-odo'"